devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy

```diff
@@ -64,6 +64,7 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008", outClusters: "0019", manufacturer: "Aurora", model: "WallDimmerMaster", deviceJoinName: "Aurora Smart Rotary Dimmer"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 0702, 0B05", outClusters: "0019", manufacturer: "sengled", model: "E1A-AC2", deviceJoinName: "Sengled DownLight"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 1000", outClusters: "0019", manufacturer: "MLI", model: "ZBT-DimmableLight", deviceJoinName: "Müller Licht Tint Bulb Dimming"
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, FC0F", outClusters: "0019", manufacturer: "OSRAM", model: " LIGHTIFY Edge-lit flushmount", deviceJoinName: "LEDVance 15\" Adjustable White Edge LitFlush Mount"
```

Comment:
This join name is what is on the ticket but I don't think it's correct. I asked for clarification on the ticket so let's not merge this until we confirm the join name.

Reply:
@tpmanley Any update here? :)

Reply:
there are conflicts now and there's a space before the model name that probably should not be there

Reply:
Per the ticket please change the join name to "SYLVANIA SMART+ Flush Mount"

Reply:
Done :)